config/jobs/kubernetes-security/genjobs.go

```diff
@@ -260,6 +254,16 @@ func convertJobToSecurityJob(j *config.Presubmit, dropLabels sets.String, defaul
 				},
 			},
 		)
+
+		// ensure decorated jobs upload to the private bucket
+		if j.Decorate {
+			j.DecorationConfig.GCSConfiguration.Bucket = "kubernetes-security-prow"
+		}
+
+		// de-dupe default utility images config so verify checks pass
```

Comment:
this was necessary to fix `hack/verify-config.sh`, it also nets us a smaller files so :man_shrugging: